grpc-interface-support/src/python.py

```diff
@@ -0,0 +1,114 @@
+# Copyright (c) 2023 Contributors to the Eclipse Foundation
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import glob
+import os
+import shutil
+import subprocess
+from pathlib import Path
+
+import proto
+from generator import GrpcInterfaceGenerator
+from util import replace_in_file, to_camel_case
+from util.templates import CopySpec, copy_templates
+from velocitas_lib import get_package_path, get_workspace_dir
+
+
+def get_required_sdk_version_python() -> str:
+    sdk_version: str = "0.11.0"
```

Comment:
THis is the default version in case there is no dependency to the SDK yet - the current state of the template does not have this simple dependency but still a link. For this PR to be released before, I have added a default version. If there is a dependency to the SDK available it will overwrite this default.